src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -279,6 +282,7 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
             } catch (error) {
                 handleError(error);
             } finally {
+                this._isBatchProcessing = true;
                 if (this.shouldDisplayExpiredAuthorizationWarning()) {
                     this.displayExpiredAuthorizationWarningModalForProcessAndDryRun(
                         async () => {
```

Comment:
Now that we aren't navigating away from this page on batch process, I think we need to explicitly close this modal - so this should be: 

```
async () => {
    await this.processBatch();
    this.dispatchEvent(new CustomEvent('closemodal')); 
}
```

Reply:
Updated!